File: src/main/java/org/example/bookfd/domain/Photo.java
package org.example.bookfd.domain;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

import javax.persistence.*;

@Getter
@Setter
@AllArgsConstructor
@NoArgsConstructor
@Entity
@Table(name = "photo")
public class Photo {

    @Id
    private int id;
    private boolean verification;
    private String name;
    private String res;
    private String path;
}

File: src/main/java/org/example/bookfd/domain/Photo.kt
package org.example.bookfd.domain

import lombok.AllArgsConstructor
import lombok.Getter
import lombok.NoArgsConstructor
import lombok.Setter
import javax.persistence.Entity
import javax.persistence.Id
import javax.persistence.Table

@Getter
@Setter
@AllArgsConstructor
@NoArgsConstructor
@Entity
@Table(name = "photo")
class Photo {
    @Id
    var id = 0
    var verification = false
    var name: String? = null
    var res: String? = null
    var path: String? = null
}
